src/test/java/com/portatlas/ServerTest.java

```diff
@@ -0,0 +1,115 @@
+package com.portatlas;
+
+import com.portatlas.request.Request;
+import com.portatlas.response.Response;
+import com.portatlas.response.StatusCodes;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+import static org.junit.Assert.*;
+
+public class ServerTest {
+
+
+    private Server server;
+    private Request getRootRequest = new Request("GET", "/" , "HTTP/1.1");
+    private Response OkResponse = Response.builder()
+            .statusCode(StatusCodes.OK)
```

Comment:
The spacing for these chained methods seem inconsistent. I'd say pick one between lining it up below the dot in `.builder()` or here